turbo/generators/templates/component/generator.ts

```
@@ -0,0 +1,149 @@
+
+          const initializer = COMPONENT_LINKS.getInitializerIfKindOrThrow(
+            SyntaxKind.ArrayLiteralExpression
+          )
+
+          const elements = initializer
+            .getElements()
+            .map((element) => element.asKind(SyntaxKind.ObjectLiteralExpression))
+            .filter((obj): obj is NonNullable<typeof obj> => obj !== undefined)
+            .map((obj) => {
+              const href = (obj.getProperty('href') as PropertyAssignment)
+                .getInitializer()!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)